src/renderable.jl

```diff
@@ -151,5 +151,52 @@ function add_renderable!(
         va = VelocityArrow(entity=entity, color=color)
         add_renderable!(rendermodel, va)
     end
-    rendermodel
+    return rendermodel
 end
+
+
+"""
+Convenience function for rendering a scene. Takes care of initializing a `RenderModel` and updating the camera.
+For full control, use `render!(rendermodel, renderables)` instead.
+"""
+function render(
+    # TODO: what about the roadway?
+    #  A) only allow rendering with roadway (as a first or second argument), scene without a roadway is not needed
+    #  B) implement a second, separate method?
+    #  C) allow roadway to be Union{Nothing, Roadway}?
+    #  D) more options??
+    scene::Frame{E}, overlays=[];
+    camera_zoom::Float64 = 10.,
+    camera_center::VecE2 = VecE2(0., 0.),
+    camera_rotation::Float64 = 0.,
+    camera_motion::Camera = SceneFollowCamera(),
+    canvas_width::Int=DEFAULT_CANVAS_WIDTH,
+    canvas_height::Int=DEFAULT_CANVAS_HEIGHT,
+    surface::CairoSurface = CairoSVGSurface(IOBuffer(), canvas_width, canvas_height)
+) where {E<:Entity}
+    rendermodel = RenderModel(
+        camera_center=camera_center, camera_zoom=camera_zoom, camera_rotation=camera_rotation
+    )
+    update_camera!(rendermodel, camera_motion, scene)
+
+    scene_renderables = []
+    for entity in scene
+        color = RGB(rand(), rand(), rand())  # TODO: random colors?? if not, how to determine id? 
```

Comment:
Not sure if random colors is a sensible default here... an alternative would be to color-code by ID, but we'd need to know what is the ego_id. Should we just assume ego_id = 1?

Reply:
I don't think we need that (see earlier comment)
also I don't think we should assume ego_id = 1 as the id might not even be an int.

Reply:
Agree on what you say about `ego_id == 1`, but then if we don't use random colors what could we do instead?